air/src/proof/mod.rs

```diff
@@ -99,28 +95,29 @@ impl Proof {
 
     // SECURITY LEVEL
     // --------------------------------------------------------------------------------------------
-    /// Returns security level of this proof (in bits).
+    /// Returns security level of this proof (in bits) using conjectured security.
     ///
-    /// When `conjectured` is true, conjectured security level is returned; otherwise, provable
-    /// security level is returned. Usually, the number of queries needed for provable security is
-    /// 2x - 3x higher than the number of queries needed for conjectured security at the same
-    /// security level.
-    pub fn security_level<H: Hasher>(&self, conjectured: bool) -> u32 {
-        if conjectured {
-            get_conjectured_security(
-                self.context.options(),
-                self.context.num_modulus_bits(),
-                self.trace_info().length(),
-                H::COLLISION_RESISTANCE,
-            )
-        } else {
-            get_proven_security(
-                self.context.options(),
-                self.context.num_modulus_bits(),
-                self.trace_info().length(),
-                H::COLLISION_RESISTANCE,
-            )
-        }
+    /// This is the conjecture on the security of the Toy problem (Conjecture 1)
+    /// in https://eprint.iacr.org/2021/582.
+    pub fn security_level_conjectured<H: Hasher>(&self) -> ConjecturedSecurity {
```

Comment:
nit: I would rename this to just `conjectured_security()`.

Reply:
changed